app/modules/user/deleteUser/deleteUser.query.js

```diff
@@ -0,0 +1,11 @@
+const { User } = require('../../../../models');
+
+const deleteUserQuery = async (userId) => {
+    return User.destroy({
```

Comment:
in certain cases we'd delete by updating a custom field in the table 'is_deleted',
varies with applications